Makefile

```diff
@@ -1,121 +1,99 @@
+BASE_DIR=build/xc7/archs/artix7/devices
 
-DEVICE=xc7a50t-basys3-test
-ARCH_TIMING_XML=$(SYMBIFLOW_ARCH_DEFS)/build/$(BASE_DIR)/xc7a50t-basys3-roi-virt/arch.timing.xml
-RR_GRAPH=$(SYMBIFLOW_ARCH_DEFS)/build/$(BASE_DIR)/rr_graph_xc7a50t-basys3_test.rr_graph.real.xml
-VPR_PARAMS=--min_route_chan_width_hint 100 \
-           --max_criticality 0.0 --max_router_iterations 500 \
-           --routing_failure_predictor off --router_high_fanout_threshold -1 \
-           --constant_net_method route --route_chan_width 500 \
-           --clock_modeling route --place_algorithm bounding_box \
-           --enable_timing_computations off --allow_unrelated_clustering on \
-           --clustering_pin_feasibility_filter off --disable_check_route on \
-           --strict_checks off --allow_dangling_combinational_nodes on \
-           --disable_errors check_unbuffered_edges:check_route
-
-# FASM
-GENFASM_BIN?=$(CONDA_DIR)/bin/genfasm
+#TARGET?=xc7a50t-arty-swbut
+TARGET?=xc7a50t-basys3
+
+DEVICE=$(TARGET)-test
+
+# Is this target outputting boths?
+ARCH_TARGET?=file_xc7_archs_artix7_devices_$(TARGET)-roi-virt_synth_tiles_pinmap.csv
+ARCH_TIMING=$(SYMBIFLOW_ARCH_DEFS)/$(BASE_DIR)/$(TARGET)-roi-virt/arch.timing.xml
+ARCH_PINMAP=$(SYMBIFLOW_ARCH_DEFS)/$(BASE_DIR)/$(TARGET)-roi-virt/synth_tiles_pinmap.csv
+
+# RRGRAPH
+RR_GRAPH_TARGET?=artix7_$(TARGET)_test_rrxml_real
+RR_GRAPH?=$(SYMBIFLOW_ARCH_DEFS)/$(BASE_DIR)/rr_graph_$(TARGET)_test.rr_graph.real.xml
 
 # Convert FASM to frames
-FASM2FRAMES=PYTHONPATH=$(CONDA_DIR)/lib/python3.7/site-packages:$(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray:$(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray/third_party/fasm python3 $(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray/utils/fasm2frames.py
+FASM2FRAMES=PYTHONPATH=$(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray:$(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray/third_party/fasm python3 $(SYMBIFLOW_ARCH_DEFS)/third_party/prjxray/utils/fasm2frames.py
 
-# XC patch
-XC7PATCH=$(SYMBIFLOW_ARCH_DEFS)/build/third_party/prjxray/tools/xc7patch
+# Place and route parameters, some are just added to speedup the computation
+VPR_PARAMS=--min_route_chan_width_hint 100 --max_criticality 0.0 --max_router_iterations 500 --routing_failure_predictor off --router_high_fanout_threshold -1 --constant_net_method route --route_chan_width 500 --clock_modeling route --place_algorithm bounding_box --enable_timing_computations off --allow_unrelated_clustering on --clustering_pin_feasibility_filter off --disable_check_route on --strict_checks off --allow_dangling_combinational_nodes on --disable_errors check_unbuffered_edges:check_route
 
-# Flashing. Just use good 'ol openocd. Everyone should've installed that on their machine.
-OPENOCD_BIN?=$(CONDA_DIR)/bin/openocd
-DIGILENT_CONFIG=openocd-digilent-basys3.cfg
+GENFASM_PARAMS=--route_chan_width 500
 
-all: top.bit
+################ Current project files and target ################
+
+# Maps the FPGA IOs to the top module verilog symbols.
+PCF=arty7.pcf
 
-$(MAP):
```

Comment:
You've removed the `$(MAP)` and `$(RR_GRAPH)` building here again - is this intentional ?

Reply:
(open?)